README.md

```diff
@@ -14,7 +14,7 @@ Scattering transforms are translation-invariant signal representations implement
 Use Kymatio if you need a library that:
 * supports 1-D, 2-D, and 3-D wavelets,
 * integrates wavelet scattering in a deep learning architecture, and
-* runs seamlessly on CPU and GPU hardware.
+* runs seamlessly on CPU and GPU hardware, with many deep learning API.
```

Comment:
too vague

Reply:
Thanks for the comment! I believe your looking at an outdated version of the README however.  I replaced this phrase in a recent commit with "with major deep learning APIs (PyTorch and TensorFlow)." per Jokiam Anden's comments. 

Resolve if that sounds fine with you.

Reply:
I'm ok with it.